docs/source_config_options.rst

```diff
@@ -1,34 +1,41 @@
 JDBC Source Configuration Options
 ---------------------------------
 
+Database
+^^^^^^^^
+
 ``connection.url``
   JDBC connection URL for the database to load.
 
   * Type: string
-  * Default: ""
   * Importance: high
+  * Dependents: ``table.whitelist``,``table.blacklist``
```

Comment:
Some of this formatting doesn't seem to work properly, see screenshot attached.
![screen shot 2016-10-10 at 3 03 37 pm](https://cloud.githubusercontent.com/assets/47366/19252653/d0bf876c-8efa-11e6-8c0a-0ccc35fbee8b.png)